src/toolbars/toolbar-component.js

```diff
@@ -52,6 +52,7 @@ angular.module('patternfly.toolbars').component('pfToolbar', {
 
     function viewSelected (viewId) {
       ctrl.config.viewsConfig.currentView = viewId;
+      ctrl.config.isTableView = viewId === 'tableView';
```

Comment:
What if the toolbar is associated with a view that is always a table view? There would be no viewConfig.